src/libs/ReportPrimaryActionUtils.ts

```diff
@@ -170,19 +229,53 @@ function getPrimaryAction(
         return CONST.REPORT.PRIMARY_ACTIONS.EXPORT_TO_ACCOUNTING;
     }
 
-    if (isRemoveHoldAction(report, reportTransactions)) {
-        return CONST.REPORT.PRIMARY_ACTIONS.REMOVE_HOLD;
+    if (isMarkAsCashAction(report, reportTransactions, violations, policy)) {
+        return CONST.REPORT.PRIMARY_ACTIONS.MARK_AS_CASH;
+    }
+
+    return '';
```

Comment:
If I understand correctly policy is optional so we could maybe add an early return here with '' when policy is undefined?

Reply:
Not really, policy will be undefined for 1:1 IOU reports but some actions are still applicable for such reports